backend/kernelCI_app/queries/notifications.py

```diff
@@ -596,3 +598,32 @@ def kcidb_tests_results(
         # dict_fetchall has a performance impact and this query returns many rows,
         # so they shouldn't be together
         return dict_fetchall(cursor=cursor)
+
+
+def get_issues_summary_data(*, checkout_ids: list[str]) -> list[dict]:
+    query = f"""
```

Comment:
Just to make sure, you know...
```suggestion
def get_issues_summary_data(*, checkout_ids: list[str]) -> list[dict]:
    if not checkout_ids:
        return []

    query = f"""
```